exercises/practice/custom-set/custom_set_test.rb

```diff
@@ -178,105 +178,105 @@ def test_adding_an_existing_element_does_not_change_the_set
     expected = CustomSet.new []
-    assert_equal expected, set2.intersection(set1)
+    assert_equal expected, set_2.intersection(set_1)
   end
 
   def test_intersection_of_two_sets_with_shared_elements_is_a_set_of_the_shared_elements
     skip
-    set1 = CustomSet.new [1, 2, 3, 4]
-    set2 = CustomSet.new [3, 2, 5]
+    set_1 = CustomSet.new [1, 2, 3, 4]
+    set_2 = CustomSet.new [3, 2, 5]
     expected = CustomSet.new [2, 3]
-    assert_equal expected, set2.intersection(set1)
+    assert_equal expected, set_2.intersection(set_1)
   end
 
   def test_difference_of_two_empty_sets_is_an_empty_set
     skip
-    set1 = CustomSet.new []
-    set2 = CustomSet.new []
+    set_1 = CustomSet.new []
+    set_2 = CustomSet.new []
     expected = CustomSet.new []
-    assert_equal expected, set1.difference(set2)
+    assert_equal expected, set_1.difference(set_2)
   end
 
   def test_difference_of_empty_set_and_non_empty_set_is_an_empty_set
     skip
-    set1 = CustomSet.new []
-    set2 = CustomSet.new [3, 2, 5]
+    set_1 = CustomSet.new []
+    set_2 = CustomSet.new [3, 2, 5]
     expected = CustomSet.new []
-    assert_equal expected, set1.difference(set2)
+    assert_equal expected, set_1.difference(set_2)
   end
 
   def test_difference_of_a_non_empty_set_and_an_empty_set_is_the_non_empty_set
     skip
-    set1 = CustomSet.new [1, 2, 3, 4]
-    set2 = CustomSet.new []
+    set_1 = CustomSet.new [1, 2, 3, 4]
+    set_2 = CustomSet.new []
     expected = CustomSet.new [1, 2, 3, 4]
-    assert_equal expected, set1.difference(set2)
+    assert_equal expected, set_1.difference(set_2)
   end
 
   def test_difference_of_two_non_empty_sets_is_a_set_of_elements_that_are_only_in_the_first_set
     skip
-    set1 = CustomSet.new [3, 2, 1]
-    set2 = CustomSet.new [2, 4]
+    set_1 = CustomSet.new [3, 2, 1]
+    set_2 = CustomSet.new [2, 4]
     expected = CustomSet.new [1, 3]
-    assert_equal expected, set1.difference(set2)
+    assert_equal expected, set_1.difference(set_2)
   end
 
   def test_union_of_empty_sets_is_an_empty_set
     skip
-    set1 = CustomSet.new []
-    set2 = CustomSet.new []
+    set_1 = CustomSet.new []
+    set_2 = CustomSet.new []
     expected = CustomSet.new []
-    assert_equal expected, set1.union(set2)
+    assert_equal expected, set_1.union(set_2)
   end
 
   def test_union_of_an_empty_set_and_non_empty_set_is_the_non_empty_set
     skip
-    set1 = CustomSet.new []
-    set2 = CustomSet.new [2]
+    set_1 = CustomSet.new []
+    set_2 = CustomSet.new [2]
     expected = CustomSet.new [2]
-    assert_equal expected, set1.union(set2)
+    assert_equal expected, set_1.union(set_2)
   end
 
   def test_union_of_a_non_empty_set_and_empty_set_is_the_non_empty_set
     skip
-    set1 = CustomSet.new [1, 3]
-    set2 = CustomSet.new []
+    set_1 = CustomSet.new [1, 3]
+    set_2 = CustomSet.new []
     expected = CustomSet.new [1, 3]
-    assert_equal expected, set1.union(set2)
+    assert_equal expected, set_1.union(set_2)
```

Comment:
The position is not important, so this could be reported, but preference is to make the position less important via communication of style. `set1, set2` does this for us, while `set_1` indicates that there is an importance of placement.

Since order does not matter  for union, we should play down the position, using it only for "uniqueness of name".